libmscore/layout.cpp

```diff
@@ -4371,8 +4371,8 @@ void Score::doLayoutRange(const Fraction& st, const Fraction& et)
             _systems.clear();
             qDeleteAll(pages());
             pages().clear();
-            LayoutContext lc(this);
-            lc.getNextPage();
+            LayoutContext lcl(this);
```

Comment:
Actually maybe it would be better not to create new `LayoutContext` here at all, the one created on line 4362 would be enough. Didn't notice that yesterday, and that pull request was create when there wasn't a layout context declared there.

Reply:
fixed